modules/UI/shared_video/SharedVideo.js

```diff
@@ -282,6 +282,7 @@ export default class SharedVideoManager {
 
             thumb.setDisplayName('YouTube');
             VideoLayout.addRemoteVideoContainer(self.url, thumb);
+            VideoLayout.resizeThumbnails(false, true);
```

Comment:
Was this related in any way to the jquery update or was it a separate bug?

Reply:
It's a bug with the jquery update that most of the comments in this PR are about.

Reply:
More specifically it's the implementation that I settled on to resolve YouTube thumbnails not displaying. More details are in the commit message but in short: I failed and used a workaround.